flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/table/stringexpr/GroupWindowStringExpressionTest.scala

```diff
@@ -47,19 +47,23 @@ class GroupWindowStringExpressionTest extends TableTestBase {
         myCountFun('string),
         'int.sum,
         weightAvgFun('long, 'int),
-        weightAvgFun('int, 'int) * 2)
+        weightAvgFun('int, 'int) * 2,
+        'w.start,
+        'w.end)
 
     // String / Java API
     val resJava = t
-      .window(JSlide.over("4.hours").every("2.hours").on("rowtime").as("w"))
+      .window(JSlide.over("4.hours").every("2.hours").on("rowtime") as 'w)
```

Comment:
Why are you changing `"w"` to a Scala expression `'w`? The test should check that Strings are correctly translated to expressions .